cunumeric/deferred.py

```diff
@@ -1132,6 +1132,33 @@ def dot(self, src1, src2, stacklevel=0, callsite=None):
                 f"dot between {rhs1_array.ndim}d and {rhs2_array.ndim}d arrays"
             )
 
+    # Create array from input array and indices
+    @profile
+    @auto_convert([1])
+    @shadow_debug("chose", [1])
```

Comment:
typo: should be "choose" instead of "chose"

Reply:
thank you!